components/global/product_switcher_menu/product_switcher_menu.tsx

```diff
@@ -54,7 +55,7 @@ class ProductSwitcherMenu extends React.PureComponent<Props> {
     }
 
     render() {
-        const {currentUser, isMessaging, isMobile} = this.props;
+        const {currentUser, isMessaging, isMobile, onClick} = this.props;
```

Comment:
Nobody appears to be passing this onClick. Is there more to this PR? How does this work?

Reply:
i am sry ... this has been lost due to a bad merge with master ... I re-added the missing functionality